src/patternfly/components/ModalBox/examples/modal-box-sm-example.hbs

```diff
@@ -0,0 +1,21 @@
+{{#> modal-box modal-box--modifier="pf-m-sm" modal-box--attribute='aria-labelledby="modal-lg-title" aria-describedby="modal-lg-description"'}}
+  {{#> modal-box-close}}
+    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Close"'}}
+      {{> button-icon button-icon--type="times"}}
+    {{/button}}
+  {{/modal-box-close}}
+  {{#> modal-box-header}}
+    {{#> modal-box-header-title  modal-box-header-title--type="h2" modal-box-header-title--attribute='id="modal-lg-title"'}}
+      Modal Header
+    {{/modal-box-header-title}}
+  {{/modal-box-header}}
+  {{#> modal-box-body modal-box-body--attribute='id="modal-lg-description"'}}
```

Comment:
Might be good to use modal-sm-description for the id in here instead of the lg references?

Reply:
doh! good catch, updated.